src/components/SiteAlert/SiteAlert.tsx

```diff
@@ -23,19 +25,33 @@ export const SiteAlert = ({
     'is-active': isActive
   });
 
+  const [isExternal, setExternal] = useState(false);
+
+  // Ensures 'window' is present
+  useEffect(() => {
+    if (url) {
+      setExternal(isExternalLink(url));
+    }
+  }, []);
+
   const tabIndex = isActive ? 0 : -1;
```

Comment:
I've not seen it mentioned in the audit but it's worth noting that the SiteAlert close button and link are still accessible from the VoiceOver rotor (see https://webaim.org/articles/voiceover/#navigation for further info on accessing that if you're not familiar with it). Should be able to fix that by adding `aria-hidden` to them in a similar way to how `tabIndex` is applied e.g.

```
const ariaHidden = isActive ? false : true;
```